tests/test_factory.py

```diff
@@ -185,6 +185,13 @@ def test_validate_fails():
     assert Factory.validate(content) == {"errors": [expected], "warnings": []}
 
 
+def test_validate_multiple_readme_files():
```

Comment:
I would also like to see a test here for mixed README formats -- in order to exercise your error check above.